lib/fat_zebra/request.rb

```diff
@@ -49,7 +49,7 @@ def delete(params = {})
 
     def initialize(params = {})
       @params = params
-      @http   = Net::HTTP.new(uri.host, uri.port, uri_proxy.host, uri_proxy.port)
+      @http   = Net::HTTP.new uri.host, uri.port
```

Comment:
Minor but not sure about removing the parens. We don't currently use this style and if we're going to adopt it we should decide as a team to do so otherwise we'll end up with a mix and match.

Reply:
is it possible to keep proxy as a param and by default using the env?